### ISSUE TITLE
Make Scholia profiles citable

### ISSUE THREAD

**Issue by Daniel-Mietchen** [fnielsen/scholia#919]
In case someone uses Scholia as part of their research, they might want to cite a given profile, and we should provide them with a mechanism to do so, e.g. a "Cite this Scholia profile" button.

As Scholia uses information from Wikidata, Wikipedia, Commons and Wembedder and processes it in various ways, caching a given page with all its embedded and JavaScript-modified content seems difficult, though not impossible. 

Some options that come to mind:
- take a whole-screen screenshot, e.g. through something like [pyscreenshot](https://pypi.org/project/pyscreenshot/)
- download the results of the embedded queries and upload them to some place like Zenodo, through which they could be cited

Ideally, this would work for both humans or machines, but initially, we should probably presume humans to be the main users.

Also, while we have a "How do I cite Scholia?" section in the FAQ, this might be something to reconsider in this context.

**Comment by jhpoelen** [comment]
@Daniel-Mietchen perhaps a way to go forward is to use [Preston](https://preston.guoda.bio) https://github.com/bio-guoda/preston . to track the provenance of the scholia queries and relate them to associated derived resources (e.g., html page, csv table). On tracking the origin and derived work of some results in a systematic and content-agnostic way, you can cite a specific work (e.g., a scholia page) without having to worry about where the Preston archive is kept. 

For example, see https://jhpoelen.nl/bees .

**Comment by jhpoelen** [comment]
So, imagine a scholia page, having a cite string similar to:

A biodiversity dataset graph: https://jhpoelen.nl/bees. 2020. [hash://sha256/85138e506a29fb73099fb050372d8a379794ab57fe4bfdf141743db0de2b985c](https://jhpoelen.nl/bees/data/85/13/85138e506a29fb73099fb050372d8a379794ab57fe4bfdf141743db0de2b985c)

for instance:
Scientific works and co-authors related to Marie Curie as rendered by Scholia.org . 2022. https://scholia.org/example/marie-curie/  hash://sha256/1234 doi://10.123/345

**Comment by Daniel-Mietchen** [comment]
@jhpoelen Yes, that sounds like a direction worth exploring. Perhaps we can demo this via the Scholia work profile for the Preston paper: https://scholia.toolforge.org/work/Q60300062 .

**Comment by Daniel-Mietchen** [comment]
@jhpoelen Note that all of the SPARQL queries used in Scholia pages are indexed in https://github.com/WDscholia/scholia/blob/master/scholia/app/templates , e.g. https://github.com/WDscholia/scholia/blob/master/scholia/app/templates/work_list-of-authors.sparql for the author list in a work profile.

**Comment by Daniel-Mietchen** [comment]
Pinging @larsgw

**Comment by larsgw** [comment]
Sounds good. Some questions:

> Scientific works and co-authors related to Marie Curie as rendered by Scholia.org

Would this be a pre-defined text for each aspect, or would something like this be easier:

> Author aspect of Marie Curie as rendered by Scholia.org

---

> hash://sha256/1234

No hashes yet until it works on dumps I suppose?

---

> Perhaps we can demo this via the Scholia work profile for the Preston paper: https://scholia.toolforge.org/work/Q60300062 .

In the interface, how would the buttons to cite the scholia page and export citations for the work be distuingished? Rename "Cite" to "Export citation" and add a new "Cite"?
